Macropad_Hotkeys/macros/linux-firefox.py

```diff
@@ -19,10 +19,10 @@
         (0x000040, 'Home', [Keycode.CONTROL, 'h']),
         (0x000040, 'Private', [Keycode.CONTROL, Keycode.SHIFT, 'p']),
         # 4th row ----------
-        (0x101010, 'Ada', [Keycode.CONTROL, 't', -Keycode.CONTROL,
+        (0x101010, 'Ada', [[Keycode.CONTROL, KEYCODE.N],
                            'www.adafruit.com\n']),   # adafruit.com in a new tab
         (0x000040, 'Dev Mode', [Keycode.F12]),    # dev mode
-        (0x101010, 'Digi', [Keycode.CONTROL, 't', -Keycode.CONTROL,
+        (0x101010, 'Digi', [[Keycode.CONTROL, KEYCODE.N],
```

Comment:
This should be `KEYCODE.T` to match the comments and prior code.

Reply:
Good catch, updated, thanks!